### ISSUE TITLE
A little help :)

### ISSUE THREAD

**Issue by SanemElbasi** [m-wrzr/populartimes#19]
Hi, first of all thank you for such a good work. I have been trying to use it however i think I am doing something wrong because when I try to execute below code it looks forever (I have left it over night) and does not give a result. If you can take a look at tell me what I am doing wrong I would be glad. Also I have one more question when we execute populartimes.get how many request are made? 

bound_lower = (40.702195412680979, -73.94647996722307)
bound_upper = (40.705036825361958, -73.94273259403137)
result = populartimes.get(api_key, "cafe", bound_lower, bound_upper, False)

**Comment by m-wrzr** [comment]
Hey,
so, there are a fey problems with your .get() call

- the place type is a list, so you should provide `["cafe"]`
- if you add only False as a parameter, you actually set `n_threads=False`. this variable should be an integer and generally set with a keyword like `n_threads=10` and as a positional argument.
- if its not running fast enough, you could try to increase the number of threads, as shown below.

so in summary, try:
```
bound_lower = (40.702195412680979, -73.94647996722307)
bound_upper = (40.705036825361958, -73.94273259403137)
result = populartimes.get(api_key, ["cafe"], bound_lower, bound_upper)
```
with `n_threads = 5` if it is still too slow for you.

**Comment by SanemElbasi** [comment]
Thank you so much. I am new to python so I am having some difficulty.
